### DISCUSSION TITLE
TP LINK TL-WN822N not work for Monterey 12.6.8

### DISCUSSION THREAD

**Discussion by SNnT23** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#311] [General]
Please fix this for TL-WN822N

**Comment by chris1111** [comment]
I have TL-WN822N it work from Catalina 10.15 to Sonoma 14

See Page about TL-WN822N you need the `V-4 / V-5 / New V-6`

![IMG_3358](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/6248794/55f72f8a-ca3a-484f-9b23-f6155a4e5849)

![IMG_3359](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/6248794/52f92d92-efcc-4077-ba12-517beec35156)

**Reply by SNnT23** [reply]
How to find download link? Bro
My us version 6

**Reply by SNnT23** [reply]
I don't know how to install. I am first user. Bro

**Comment by chris1111** [comment]
V-6 suppose to work, try diferant USB port

**Reply by SNnT23** [reply]
Ok I try now. It's Download link here. Yes?

**Reply by SNnT23** [reply]
Hey I got this. Thank you a lot, bro.
